src/bilder/components/pomerium/files/config.yaml.j2

```diff
@@ -0,0 +1,41 @@
+{{with secret "secret-data/airbyte/pomerium" }}
+# See detailed configuration settings : https://www.pomerium.com/docs/reference/
+
+# this is the domain the identity provider will callback after a user authenticates
+# form: https://< whatever >
+authenticate_service_url: "{{ .Data.authenticate_server_url }}"
+
+# certificate settings:  https://www.pomerium.com/docs/reference/certificates.html
+# These paths are within the docker container, not the host machine
+certificate_file: /pomerium/cert.pem
+certificate_key_file: /pomerium/privkey.pem
+
+##################################################################################
+# Identity provider settings : https://www.pomerium.com/docs/identity-providers/ #
+# The keys required in this section vary depending on your IdP. See the          #
+# appropriate docs for your IdP to configure Pomerium accordingly.               #
+##################################################################################
+# https://www.pomerium.com/docs/identity-providers/github.html
+idp_provider: github
+idp_client_id: "{{ .Data.idp_client_id }}"
+idp_client_secret: "{{ .Data.idp_client_secret }}"
+idp_service_account: "{{ .Data.idp_service_account }}"
+
+# https://github.com/pomerium/pomerium/blob/main/scripts/generate_self_signed_signing_key.sh
+signing_key: "{{ .Data.signing_key }}"
+
+# Generate 256 bit random keys  e.g. `head -c32 /dev/urandom | base64`
+cookie_secret: "{{ .Data.cookie_secret }}"
+
+routes:
+- from: "{{ .Data.authenticate_server_url }}/"
```

Comment:
Is there a way to set up a non-authenticated route that we can use for communication from Dagster to Airbyte?

Reply:
I think we would probably need to put another layer in front of things to accomplish that. Which, tbh, is probably the appropriate thing to do anyways. 
```
Internet -> Nginx/Caddy -> (User) -> Pomerium -> Github -> Pomerium -> Airbyte
                  \->(Special URL or Path) -> (Dagster + AuthToken)---/^
```

I stand and accuse you of scope creep. Unless that is written down in the requirements that I never actually read. :-)

Reply:
![ok](https://media.giphy.com/media/QYwB8ai7mtORGRAxJZ/giphy.gif)

Reply:
:rofl: